### PR TITLE
Adding Utils.deep_symbolize_keys

### PR DESCRIPTION
**Pull request by camertron**
Convenience method used for the queuing work I'm doing.

@lumoslabs/platform

### CODE REVIEW

File: txgh/lib/txgh/utils.rb [hunk 1]
```diff
@@ -64,6 +64,23 @@ def booleanize(obj)
           obj
       end
     end
+
+    def deep_symbolize_keys(obj)
```
Comment (jdoconnor) [inline txgh/lib/txgh/utils.rb:68]: are we using activesupport anywhere yet? this is defined there if we are.

**Review by jdoconnor — COMMENTED** [review]
just a question that may simplify this

File: txgh/lib/txgh/utils.rb (hunk 1 above)
Comment (camertron) [reply txgh/lib/txgh/utils.rb:68]: Ok, I did a cursory read-through of all the code in txgh and I don't think we're using any activesupport-ish functionality (except `deep_symbolize_keys`). Txdb uses the activesupport inflector and accordingly depends on it in the gemspec.

**Comment by jdoconnor** [comment]
+1

**Pull request merged**